services/terrain/src/terrain/services/permanent_id_requests.clj

```diff
@@ -355,4 +379,5 @@
   (let [identifier (complete-permanent-id-request (:shortUsername current-user)
                                                   (admin-get-permanent-id-request request-id nil))]
     (update-permanent-id-request request-id nil (json/encode {:status       status-code-completion
+                                                              :comments     identifier
```

Comment:
This seems unrelated (though presumably fine) -- what's going on here adding the extra key with the same info?